components/Activity/ActivityBox.tsx

```diff
@@ -0,0 +1,114 @@
+import { GraphQLResult, GRAPHQL_AUTH_MODE } from "@aws-amplify/api/lib/types"
+import { API } from "aws-amplify"
+import moment from "moment"
+import React, { useEffect, useState } from "react"
+import { Text, TouchableHighlight, View } from "react-native"
+import { Activity } from "src/API"
+import { activityByGroup } from "../../src/graphql-custom/queries"
+import ActivityBoxStyles from "./ActivityBoxStyles"
+type Selected = string
+
+type Props = {
+  title: string
+  activityGroupType: string
+  activityGroupId: string
+}
+
+const ActivityBox = ({ title, activityGroupType, activityGroupId }: Props): JSX.Element => {
+  const [activities, setactivities] = useState<Array<Activity>>([])
+  const [selected, setSelected] = useState<Selected>("Today")
```

Comment:
`Selected` is equivalent to the `string` primitive, so you don't need to pass this type (and can get rid of `Selected`). 

Or perhaps you meant to do `type Selected = "Today" | "Yesterday" | "Last 7 Days"`?

Reply:
I meant to do the latter. Should be fixed now.